official/utils/flags/_performance.py

```diff
@@ -270,4 +271,9 @@ def _check_fp16_implementation(flags_dict):
             "Whether to enable tf.data's `experimental_slack` option.")
     )
 
+  if enable_xla:
+    flags.DEFINE_boolean(
```

Comment:
Maybe also have the Keras Resnet model use this version of the flag, so we don't have two definitions of the `enable_xla` flag.

Reply:
I'll follow your "hack" (pattern) and add it in.  I really want to fix the flags correctly for keras.  I do not like that we call into run_loops to make the flags.

Reply:
Yeah, the pattern is pretty ugly. It's why I used the word "Maybe" in this comment :).

Reply:
Fixed, I think it is slightly better and ready for a bigger change soon.